src/PresentationalComponents/Shield/consts.js

```diff
@@ -0,0 +1,39 @@
+export const colorList = {
+    default: 'var(--pf-global--Color--200)', // grey
+    danger: 'var(--pf-global--danger-color--100)',
+    warning: 'var(--pf-global--warning-color--100)',
+    orange: '#ec7a08' // orange
+};
+
+/* I'm not shure that the list of impacts is ordered correctly by relevancy */
+export const impactList = {
```

Comment:
Can we also support 4 levels numerically?
```
4: Critical
3: High
2: Medium
1. Low
```

Reply:
How? Like this?
`this.impactList.numerically = { 1: 'Low', 2: 'Medium', 3: 'High', 4: 'Critical' }`

Reply:
Yes. That's perfect.